en/docs/testing/unit-testing-in-build-pipeline.md

```diff
@@ -0,0 +1,28 @@
+# Unit Testing in Choreo Build Pipeline
+
+Choreo supports unit testing as part of the build pipeline, enabling users to validate their applications before deployment. This feature ensures that code changes are tested early, reducing the risk of defects in production.
```

Comment:
Suggestion:
Choreo supports integrating unit tests into the build pipeline, allowing you to validate code before deployment. This ensures early testing of code changes, reducing the risk of defects in production.